src/items/dtos/item.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { type Item } from '../entities/item.entity';
+
+export class ItemDto {
+  constructor(item: Item) {
+    this.id = item.id;
+    this.name = item.name;
+  }
+
+  id: number;
+  name: string;
+}
```

Comment:
Since this DTO will be sent back as an API response, we need to add the appropriate decorators so Swagger can render our types in the documentation:

```suggestion
export class ItemDto {
  @ApiProperty()
  public readonly id: number;
  
  @ApiProperty()
  public readonly name: string;
  
  constructor({ id, name }: Item) {
    this.id = id;
    this.name = name;
  }
}
```